Sources/Request.swift

```diff
@@ -4,14 +4,17 @@ public struct Request: Message {
     public var version: Version
     public var headers: Headers
     public var body: Body
-    public var storage: [String: Any] = [:]
+    public var cookies: [String: String]
```

Comment:
If we don't have a `cookies` property on the `Request`, then there is no way to support cross-project interoperability with any code (ex: Middleware) that wants to use cookies. The middleware would have to parse the cookies internally.

Take the following example.

```swift
class CookieMiddleware: S4.Middleware {
   func handle(...) {
       print(request.cookies["test-cookie"]
   }
}
```

The above middleware would work in a Vapor project, because Vapor adds an extension to `Request` for accessing cookies from the headers, `cookies: [String: String]`. Anyone who sees this middleware might think that they can drag and drop it into any other `S4` compatible project since it only uses `S4.Middleware` and `S4.Request`. But, if they drop it into a project that doesn't have the exact same `Request` extensions that Vapor has, it will not compile.

By moving `cookies` to `S4`, we are ensuring that whatever method of parsing cookies that is already being used by the framework does not need to be replicated in middleware that wants to use cookies. 

I believe this merits `cookies` being a stored property on `Request`.